packages/react-core/src/components/TimePicker/__tests__/TimePicker.test.tsx

```diff
@@ -72,7 +72,7 @@ describe('test isInvalid', () => {
     expect(view.find('.pf-m-error')).toHaveLength(0);
   })
 
-  test('should be invalid after onChange', () => {
+  test('should be invalid after onChange and onBlur', () => {
```

Comment:
isn't it only onBlur now?

Reply:
It's actually testing `valid` onChange and `invalid` onBlur... 🤯